floss/main.py

```diff
@@ -483,7 +496,7 @@ def print_identification_results(sample_file_path, decoder_results):
             headers=["address", "score"]))
 
 
-def print_decoding_results(decoded_strings, group_functions, quiet=False, expert=False):
+def print_decoding_results(decoded_strings, group_functions, quiet=False, expert=False, only_suspicious_strings=False):
```

Comment:
these prototypes are getting a bit out of control (not an issue specific to this PR). lots of options and lots of possible combinations.

@mr-tz:

the code can be made more succinct if we use a config object (e.g. a dict of options); however, this doesn't mitigate the number of things to track.

we may want to spend a little time thinking about the different classes of output options and see if we can refactor here. the goal to make things concise and easy to understand.

for example, there may be multiple types of filter that we do (suspicious, static strings, unique, etc.) and maybe we could configure and implement these in a similar way. probably just needs some time dedicated to this.